read me deleted as app is decommissioned (#13)
LBHackney-IT/lbh-fss-step-function: README.md

> This application and it's cloud recources has now been decommissioned

Reviewer: It may be helpful to people looking for historical context to read the readme, so I'd suggest leaving the readme visible but adding a prominent note saying it's archived, e.g:
```suggestion
# Find Support Services organisation re-verification process

> ⚠️ This application and it's cloud resources have now been decommissioned, and this repository is no longer maintained.
```

> <!--

Reviewer: ```suggestion
```
Remove comment block to reinstate text (matching the suggestion above). nit: If we're removing the text we should delete it rather than commenting it out.

Reviewer 2: This one is my bad. During last decommissioning I told them to comment out the README as the entire readme was merely a boiler plate on how to spin app up.

This suggestion I gave them before doesn't apply in this case as repository business describing portions of the README are best to be retained.

> <!--

Reviewer 2: Agreed with Ryan.

In this case, commenting out the description won't make sense as this is not the application that has been deployed, and outlived its usefulness, but rather this application is a feature that was completed but never released due to budget cuts and some turmoil.

This README documents all the business context regarding this application some of which you won't find in any document. So it's best to keep it.

## Ownership
This along with the linked services and repositories is now owned by the [@LBHackney-IT/targeted-services](https://github.com/orgs/LBHackney-IT/teams/targeted-services) team.

> 
> -->

Reviewer: ```suggestion

```
Remove comment block to reinstate text (matching the suggestion above). nit: If we're removing the text we should delete it rather than commenting it out.
